Client/index.js

```diff
@@ -2,4 +2,4 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App.jsx';
 
-ReactDOM.render(<App />, document.getElementById('app'));
+ReactDOM.render(<App />, document.getElementById('image-carousel'));
```

Comment:
should `App` be changed to `ImageCarousel`? I guess you may be doing this in the other PR

Reply:
my bad--I was looking at the wrong commit. Going back to this